lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -14,43 +14,223 @@
 
 part of 'webcrypto.dart';
 
+/// Elliptic Curve Diffie-Hellman (ECDH) is a key agreement protocol that allows
+/// two parties to establish a shared secret over an insecure channel.
+/// 
+/// An [EcdhPrivateKey] instance represents a private key that can be used to
+/// derive a shared secret with a public key.
+/// 
+/// Instances of [EcdhPrivateKey] can be imported from:
+/// * PKCS8 Key using [EcdhPrivateKey.importPkcs8Key], and,
+/// * JSON Web Key using [EcdhPrivateKey.importJsonWebKey].
+/// 
+/// A random key pair can also be generated using [EcdhPrivateKey.generateKey].
+/// 
+/// **Example**
+/// ```dart
+/// import 'package:webcrypto/webcrypto.dart';
+/// 
+/// Future<void> main() async {
+///   // Generate a new key pair using the P-256 curve.
+///   final keyPair = await EcdhPrivateKey.generateKey(EllipticCurve.p256);
+///   
+///   // Derive 256 bits from the private key using the public key.
+///   final derivedBits = await keyPair.privateKey.deriveBits(256, keyPair.publicKey);
+/// 
+///   // Export the private key as a JSON Web Key.
+///   final exportedPrivateKey = await keyPair.privateKey.exportJsonWebKey();
+/// }
+/// ```
 @sealed
 abstract class EcdhPrivateKey {
   EcdhPrivateKey._(); // keep the constructor private.
 
-  // Note. unsupported on Firefox, see EcdsaPrivateKey.importPkcs8Key
+  /// Import [EcdhPrivateKey] using the PKCS8 format[1].
+  /// The [keyData] parameter is an octet string whose 
+  /// contents are the value of the private key.
+  /// The [curve] parameter specifies the curve to use 
+  /// for the key pair.
+  /// 
+  /// **Example**
+  /// ```dart
+  /// import 'package:webcrypto/webcrypto.dart';
+  /// 
+  /// final pkcs8Key = [ /* ... */ ];
+  /// 
+  /// Future<void> main() async {
+  ///   // Import the private key from PKCS8 format.
+  ///   final privateKey = await EcdhPrivateKey.importPkcs8Key(
+  ///     pkcs8Key,
+  ///     EllipticCurve.p256,
+  ///   );
+  /// }
+  /// ```
+  /// 
+  /// [1]: https://datatracker.ietf.org/doc/html/rfc5208
   static Future<EcdhPrivateKey> importPkcs8Key(
     List<int> keyData,
     EllipticCurve curve,
   ) {
+  // Note. unsupported on Firefox, see EcdsaPrivateKey.importPkcs8Key
```

Comment:
This should be noted in a remark similar to what we have at the bottom of:
https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/encryptBytes.html

I think the "see EcdsaPrivateKey.importPkcs8Key" is just a note to us, because we probably wrote more about it there. Hopefully, the source contains a bug we can reference.

Reply:
Moved the Note above the function definition. Removed "see EcdsaPrivateKey.importPkcs8Key"

Reply:
What I meant was that we should probably note in the documentation comment that it won't work for Firefox.

If we actually lookup the code for what the comment says, we'll see:
https://github.com/google/webcrypto.dart/blob/master/lib/src/webcrypto/webcrypto.ecdsa.dart#L21-L29

That comment sends us to:
https://bugzilla.mozilla.org/show_bug.cgi?id=1133698

Which appears to be fixed 3 years ago :tada:, so maybe `importPkcs8Key` works for EC keys on Firefox. We should update our tests to see if this is the case.

If `importPkcs8Key` works, then we won't need to document that it doesn't work on Firefox.

Filed: https://github.com/google/webcrypto.dart/pull/117 to remove this comment.
As it now appears to work.